lib/cqm/converter/bonnie_measure.rb

```diff
@@ -129,6 +131,19 @@ def self.measure_and_valuesets_to_cqm(bonnie_measure, hds_valuesets)
       cqm_measure.value_sets = cqm_valuesets
       cqm_measure
     end
+
+    def self.update_oids(cql_library)
+      name_oid_hash = JSON.parse(File.read(File.join(File.dirname(__FILE__), 'name_oid_map.json')))
+      return unless cql_library['elm']['library'] && cql_library['elm']['library']['codeSystems'] && cql_library['elm']['library']['codeSystems']['def']
+
+      cql_library['elm']['library']['codeSystems']['def'].each do |codesystem|
```

Comment:
*You do not need to use this recommendation* but a slightly cleaner (more ruby solution) would be 
```
(cql_library.dig('elm', 'library', 'codeSystems', 'def') || []).each do |codesystem|
```
this will allow you to remove the return clause on line 137 because dig is a safe way to "dig" through a hash.